src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import Card from "./";
+import Col from "../Col";
+import Row from "../Row";
+
+/**
+ * This is a [React](https://reactjs.org/) component for the Vanilla [Card](https://docs.vanillaframework.io/patterns/card/).
```

Comment:
How is this transformed into the description in the story? It doesn't seem to be treated as markdown? Or can it?

Right now you seem to be inserting double <br> to make it look like 2 paragraphs, while it's all inserted into one. Which is not ideal.

Reply:
I'd recommend to move this to the component file itself, storybook will pick it up and display in using autogenerated docs.

Reply:
makes sense! moved to component, and removed the `<br/>` tags on [9179238](https://github.com/canonical/react-components/pull/1010/commits/9179238a515b9c84dc247fe6751b58a5984486b7)